<?xml version="1.0" ?>
<launch>

  <!-- Start assembly client -->
  <node name="object_database_server"
	pkg="aist_object_database" type="object_database_server.py"
	clear_params="true" output="screen">
    <rosparam param="object_descriptions"
	      file="$(find aist_fastening_tools
		    )/config/fastening_tool_properties.yaml"/>
    <rosparam param="object_descriptions/precision_tool"
    	      file="$(find aist_precision_gripper
    		    )/config/precision_gripper_properties.yaml"/>
    <rosparam param="object_descriptions"
	      file="$(find aist_description
		    )/parts/config/parts_properties.yaml"/>
  </node>

</launch>
